recipes/pymultihash/LICENSE

```diff
@@ -0,0 +1,9 @@
+MIT License
```

Comment:
Looks like this file is in the tarball, maybe all other license too. Can you remove them to avoid adding it twice? Also, if you hardcode it when it is present, you can get them out of sync with the file in the tarball in case it changes.

Reply:
Done. One really didn't have them, and I kinda forgot whether to include or not. Sorry!

Reply:
This one is included, right? So maybe remove it as well?

Reply:
thx. I think I only checked the github repo, and there the license is not present at the top. Fixed now.